src/openvpn.mli

```diff
@@ -179,9 +179,6 @@ module Config : sig
         Indirectly also by [client]. *)
 
     | Tls_key      : X509.Private_key.t k
-    (** TODO Tls_key : X509.private_key * [`Incoming|`Outgoing] k
-        --key-direction governs this for inlined files
-        see comment in {!a_key} *)
```

Comment:
FWIW, this comment is wrong, it is about `--tls-auth` where we respect the direction (incoming/outgoing) already.